### DISCUSSION TITLE
Do uploaded Screenshots just get published instantly?

### DISCUSSION THREAD

**Discussion by pct-cclausen** [fastlane/fastlane#21383] [Q&A]
I am building a CI/CD pipeline for an old app.
I've set up an internal testing track and upload the test build like this:

`upload_to_play_store(track:'internal', aab: binarypath, skip_upload_changelogs: true, metadata_path: metadatapath, package_name: appId)`

Now the screenshots of the in-development new app from the metadata show up in the app store listing of the old app for public users?

I though track internal means all data stays internal? Is this intentional? It seems like a super dangerous behavior.

**Comment by Pandfuldell1949** [comment]
Hello , @pct-cclausen I read about your problem i Just want to tell you that , I have some key point for you that help you out :

In view of the code scrap you gave, it seems as though you are transferring an Android Application Group (AAB) to the inward testing track on the Google Play Store. The inside testing track is planned for use by your inward group and analyzers to see and test the application before it is made accessible to general society. It shouldn't influence the public application posting or make the being developed application screen captures noticeable to public clients.

On the off chance that the screen captures from the being developed application are showing up on the application store posting of the old application for public clients, this conduct isn't normal and could be an expected issue. It is to be sure not purposeful and can be viewed as a hazardous way of behaving, particularly on the off chance that delicate or inadequate data is being presented to people in general.

Double-check the parameters used in the upload process to ensure that the correct track ('internal') is being targeted and that the metadata corresponds to the right app. [Myjdfaccount Login](https://www.myjdfaccount.net/)

 Check your Google Play Console settings to confirm that the internal testing track is properly configured and that the in-development app is not unintentionally linked to the old app's public listing.As a best practice, set up a test environment with a separate Google Play Console account or a test app to validate the behavior of the internal testing track before applying it to the production app.If the issue persists and you cannot identify the cause, reach out to Google Play Developer Support for assistance. They can investigate the specific case and provide guidance on resolving the issue.

I hope you like my reply Thank you.
Best , 
[ Marilyn R. Palmer ]

**Comment by Phillips235** [comment]
Manage app information. Upload app previews and screenshots. Use screenshots and app previews to visually communicate your app's user experience. [](-[Tellthebell](https://tellthebell.fun/))
